CONTRIBUTING.md

```diff
@@ -62,30 +62,31 @@ questions: what changed and why. The subject line should feature the what and
 the body of the commit should describe the why.
 
 ```
-capture: add comment for variable declaration
+capture(ticdc): add comment for variable declaration
 
 Improve documentation.
 ```
 
 The format can be described more formally as follows:
 
 ```
-<subsystem>: <what changed>
+<subsystem>[(ticdc|dm|both)]: <what changed>
```

Comment:
I don't know if this `both` is a good idea, but for now I think it might be better if we write it out explicitly. Plus it will make it easier to automate our checks later.

Reply:
@lance6716 @Ehco1996  Do you have any thoughts on it? Would you rather use`both` or write `dm,ticdc` explicitly?

Reply:
i think they are same, if we choose `both` we can add some rule like `both=ticdc,dm...` in bot